src/runtime/ContextFeed.tsx

```diff
@@ -43,11 +43,7 @@ class ContextFeed extends React.Component<Props>
             </Consumer>
           )
         } else {
-          if (values.indexOf(undefined) > -1) {
```

Comment:
Let's leave this here, as the goal is to handle loading for the user of the library. We can add the ability to disable this if the user wants, but by default I think it's better to handle it for them.

Additional thought / want: we should support the user being able to override the loading component. This can be made into another task & PR.